src/main/java/lotto/domain/rank/Rank.java

```diff
@@ -0,0 +1,38 @@
+package lotto.domain.rank;
+
+import java.util.Arrays;
+
+public enum Rank {
+
+    FIRST(6, 2_000_000_000),
+    SECOND(5, 30_000_000),
+    THIRD(5, 1_500_000),
+    FOURTH(4, 50_000),
+    FIFTH(3, 5_000),
+    NO_MATCH(0, 0);
+
+    private int matchCount;
+    private int money;
```

Comment:
private final int matchCount;
    private final int money
```suggestion
    private final int matchCount;
    private final int money;
```